src/nomad_material_processing/general.py

```diff
@@ -545,24 +572,39 @@ class SubstrateCrystalProperties(CrystalProperties):
             component=ELNComponentEnum.EnumEditQuantity,
         ),
     )
-    orientation = Quantity(
+    hkl_reciprocal = Quantity(
         type=str,
-        description="""
-        Alignment of crystal lattice with respect to a vector normal to the surface
-        specified using Miller indices.
-        """,
+        description="""Reciprocal (or Absolute) Miller indices, (hkl),
+        denote a family of planes specified an orthogonal vector in the basis of
+        the primitive reciprocal lattice vectors. """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+            label='Reciprocal Indices (hkl)',
+        ),
+    )
+    hkl_direct = Quantity(
+        type=str,
+        description="""Direct (or Relative) Miller indices, [hkl],
```

Comment:
You can remove "relative" here for the same reasons as mentioned above.
Again, here too there is a symmetry-reduced notation `<hkl>`, in case you ever want to add these.